pypeit/core/wavecal/templates.py

```diff
@@ -836,7 +859,13 @@ def main(flg):
     #flg += 2**28
 
     # Keck KCWI
-    flg += 2**29
+    #flg += 2**29
+
+    # MMT MMIRS
+    #flg += 2**30
+
+    # LBT MODS
```

Comment:
This is going to become unwieldy soon... I wonder (not this PR) if we should come up with a better way to manage the `flg` values

Reply:
agreed.